python/ql/src/Statements/UnreachableCode.ql

```diff
@@ -13,6 +13,18 @@
 
 import python
 
+private Name variableInvolvedInComparison (Compare c) {
+    result = c.getLeft().getASubExpression*() or
+    result = c.getAComparator().getASubExpression*()
+}
+
+private predicate allInvolvedVariablesAreInvolvedInPrevious(Compare c, Compare prevc) {
+    forall (Name n | 
```

Comment:
I think this should be `forex` instead of `forall`. As written currently, a `Compare c` with _no_ variables will trivially satisfy this predicate (since `variablesInvolvedInComparison(c)` will return zero results), and this seems wrong to me.

Reply:
Done